frontend/src/queries/nodes/InsightViz/insightDisplayConfigLogic.ts

```diff
@@ -59,6 +59,18 @@ export const insightDisplayConfigLogic = kea<insightDisplayConfigLogicType>([
                 }
             },
         ],
+        showPercentStackView: [
+            (s) => [s.isTrends, s.isStickiness, s.isLifecycle, s.display],
+            (isTrends, isStickiness, isLifecycle, display) => {
+                if (isTrends || isStickiness) {
+                    return display === ChartDisplayType.ActionsBar
+                } else if (isLifecycle) {
+                    return true
+                } else {
+                    return false
+                }
```

Comment:
```suggestion
                } else {
                    return isLifecycle
                } 
```

Reply:
I wonder if we can simplify even further... could it be closer to 

```
return display === ChartDisplayType.ActionsBar || isLifecycle
```

(not checked if this will work :))

Reply:
Changed to this (isTrends || isStickiness ? display === ChartDisplayType.ActionsBar : isLifecycle)